vcs/vcs2vtk.py

```diff
@@ -29,7 +29,7 @@
     "eqdc",
     "tmerc",
     "stere",
-    "lcca",
```

Comment:
At least since 2015-09-21. laea = Lambert Azimuthal Equal Area while lcca =  Lambert Conformal Conic Alternative. I see this more like an internal name (this is what proj4 uses) so I don't think it would be necessary to keep lcca around. We already have the long name: 'Lambert Azimuthal'.